pkg/sharing/sharing.go

```diff
@@ -6,14 +6,21 @@ import (
 	"github.com/cozy/cozy-stack/pkg/consts"
 	"github.com/cozy/cozy-stack/pkg/couchdb"
 	"github.com/cozy/cozy-stack/pkg/instance"
+	"github.com/cozy/cozy-stack/pkg/jobs"
 	"github.com/cozy/cozy-stack/pkg/permissions"
+	multierror "github.com/hashicorp/go-multierror"
 )
 
 const (
 	// StateLen is the number of bytes for the OAuth state parameter
 	StateLen = 16
 )
 
+// Msg is an interface for the sharing worker messages
```

Comment:
This interface looks weird (too generic?)

Reply:
I was thinking of the methods: with just one `ID` method, you can have a lots of structs that implement this interface even if they don't have anything to do with a message for a sharing worker

Reply:
Ok I see. The last commit removes this interface anyways, as it is no longer necessary by using the trigger ids in the sharing struct